CorsixTH/Lua/dialogs/menu.lua

```diff
@@ -547,7 +547,7 @@ end
 -- game sounds controls are added.
 local function audio_options(menu, game)
   local app = TheApp
-  if not app.audio.has_bg_music then return end
+  if not app:isAudioEnabled() then return end
```

Comment:
This is fine as a fix. But it makes music options available again that there isn't much point in controlling/using. Ideally per #2549 a better alternative is any music option should really be disabled instead if there is no songs.

(A separate issue is that movies are also tied to music volume.)